pxd_fastpath.c

```diff
@@ -1023,14 +799,23 @@ static int __do_bio_filebacked(struct pxd_device *pxd_dev, struct pxd_io_tracker
 
 	return ret;
 }
-
 #endif
 
-static
-int pxd_handle_io(struct thread_context *tc, struct pxd_io_tracker *head, int dir)
+static void pxd_process_fileio(struct work_struct *wi)
```

Comment:
file IO will use this routine to complete IO to the backing file. This is run through a work queue.